applications/dashboard/openapi/tick.yml

```diff
@@ -0,0 +1,32 @@
+openapi: 3.0.2
+info:
+paths:
+  /tick:
+    post:
+      summary: Register a page view for statistical and analytical purpose.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TickRequest'
+        required: false
+      responses:
+        '201':
+          description: Success
+      tags:
+      - Statistics
+components:
+  schemas:
+    TickRequest:
+      description: Both body request parameters are optional
+      properties:
+        Path:
```

Comment:
These should adhere to our naming conventions and be lower camel case.

Reply:
I was trying to keep backward compatibility with old endpoint.